### PR TITLE
docs(sdb): add doc on ssl/tls MTA-4669

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+[node-postgres](https://node-postgres.com/) doesn't support `sslmode=verify-full` and `sslrootcert=system`, but either the default connection string option `sslmode=require` or the driver-specific parameter `ssl:true` option checks for certificate validity.
+
+To ensure SSL/TLS is enforced and your server certificate is valid, add `ssl:true` to your connection parameters:  
+```js
+const client = new Client({
+  host: {host}, //Host formated as {database-id}.pg.sdb.{region}.scw.cloud
+  port: {port}, //Default port for PostgreSQL is supported: 5432
+  database: {databasename},
+  user: {username}, //IAM principal ID of the user or application your are connecting with
```
Comment (ldecarvalho-doc) [inline serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx:98]:
```suggestion
  user: {username}, //IAM principal ID of the user or application you are connecting with
```

File: serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  password: {username}, //IAM Secret Key of the user or application your are connecting with
```
Comment (ldecarvalho-doc) [inline serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx:99]:
```suggestion
  password: {username}, //IAM Secret Key of the user or application you are connecting with
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  - [Python](#pythonpsycopg2)
+  - [Python](#pythondjango)
+  - [Node.js](#nodejsnode-postgres)
+  - [Node.js](#nodejspostgresjs)
+  - [Node.js](#javajdbc)
+  - [psql](#psql)
+
+## Generic configuration settings
+
+Starting from PostgreSQL 16, you can set up SSL/TLS to rely on the default certification authority certificates trusted by your operating system. To do so, use the additional configuration parameters `sslmode=verify-full` and `sslrootcert=system`.
```
Comment (jcirinosclwy) [inline serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx:25]: Should this really be 'certification authority certificates'?

Comment (SamyOubouaziz) [reply serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx:25]: image
yes 😂

Comment (SamyOubouaziz) [reply serverless/sql-databases/api-cli/secure-connection-ssl-tls.mdx:25]: or "certificate authority", both exist (I know it sounds weird)

**Review by ofranc — APPROVED** [review]

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**